htdocs/webportal/public/controllers/orderlist.controller.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/* Copyright (C) 2017 	Laurent Destailleur		<eldy@users.sourceforge.net>
+ * Copyright (C) 2023	Lionel Vessiller		<lvessiller@open-dsi.fr>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * \file        public/controllers/orderlist.controller.php
+ * \ingroup     webportal
+ * \brief       This file is a controller for order list
+ */
+
+dol_include_once('/webportal/public/class/html.formlistwebportal.class.php');
```

Comment:
We should not have classes into the public subdirectory.
Only minimum visible front pages must be stored into the dir public.
So can you move php class files that are not front pages into /webportal/class
and move all public front pages into htdocs/public/webportal
instead of htdocs/webportal/public

All front office pages must be into htdocs/public/... like it is now, because this convention is used by several securities protections like fail2ban rules or phpunit  analysis tool knowing that htdocs/public are not backoffice pages.

Reply:
It has been done